core/src/test/java/com/nuodb/migrator/backup/globalStore/GlobalStoreTest.java

```diff
@@ -0,0 +1,187 @@
+package com.nuodb.migrator.backup.globalStore;
+
+import com.nuodb.migrator.backup.loader.BackupLoader;
+import com.nuodb.migrator.backup.loader.LoadTable;
+import com.nuodb.migrator.dump.DumpJob;
+import com.nuodb.migrator.globalStore.GlobalStore;
+import com.nuodb.migrator.jdbc.dialect.Dialect;
+import com.nuodb.migrator.jdbc.dialect.Script;
+import com.nuodb.migrator.jdbc.metadata.Column;
+import com.nuodb.migrator.jdbc.metadata.Schema;
+import com.nuodb.migrator.jdbc.metadata.Sequence;
+import com.nuodb.migrator.jdbc.metadata.Table;
+
+import com.nuodb.migrator.jdbc.metadata.generator.ScriptGeneratorManager;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import static com.nuodb.migrator.jdbc.metadata.MetaDataUtils.createTable;
+import static java.sql.Types.BIGINT;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+
+import static com.google.common.collect.Lists.newArrayList;
+import static com.nuodb.migrator.jdbc.metadata.MetaDataUtils.createSequence;
+import static com.nuodb.migrator.jdbc.session.SessionUtils.createSession;
+import static org.mockito.BDDMockito.*;
+import static org.mockito.Mockito.when;
+import static com.nuodb.migrator.jdbc.metadata.DatabaseInfos.NUODB;
+import static org.testng.Assert.*;
+
+import java.sql.DatabaseMetaData;
+import java.util.Collection;
+import java.util.Map;
+
+import static org.mockito.Mockito.mock;
+
+import com.nuodb.migrator.spec.DumpJobSpec;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import org.mockito.Spy;
+import com.nuodb.migrator.job.JobExecution;
+import com.nuodb.migrator.job.JobExecutor;
+
+import static com.google.common.collect.Maps.newHashMap;
+import static com.nuodb.migrator.job.JobExecutors.createJobExecutor;
+
+
+import com.nuodb.migrator.jdbc.dialect.NuoDBDialect;
+
+
+@SuppressWarnings("all")
```

Comment:
Why do you need this? Normally you should not need to suppress anything, unless you are bumping into annoying Java limitations having to do with generics, which makes suppression of `rawtypes` or `unchecked` unavoidable.

Otherwise, it is almost always better way to handle static warnings by fixing the code rather than suppressing them.

Reply:
Removing the SuppressWarnings annotation, Addressing specific warnings instead of suppressing them globally.